float32 range_ahead
float32 range_left
float32 range_right
float32 range_rear
~
